static_src/actions/user_actions.js

```diff
@@ -221,6 +216,57 @@ const userActions = {
     return Promise.resolve(userInfo);
   },
 
+  fetchCurrentUserRole(userGuid) {
```

Comment:
I believe this won't be needed anymore once mine gets pulled in. We could try and remove it and see if it works correctly after that?